src/format.ts

```diff
@@ -104,6 +105,23 @@ export function automaticResultTable(results: ResultStats[]): AutomaticResults {
   return {fixed: fixedTable, unfixed: unfixedTable};
 }
 
+export function collatedResultTables(results: ResultStatsWithDifferences[]) {
```

Comment:
Would there be any value in skipping the difference computation for dimensions that are partitioned? Though I guess that is larger breaking change that impacts more parts of the code.... Perhaps something to explore later?